spec/unit/installer/user_project_integrator/target_integrator_spec.rb

```diff
@@ -637,6 +612,47 @@ module Pod
         end
       end
 
+      describe 'Script paths' do
+        it 'calculates the output paths of the embed frameworks script' do
+          paths = [
+            Framework.new('${PODS_ROOT}/DebugVendoredFramework/ios/SomeFramework.framework',
+                          '${PODS_ROOT}/DebugVendoredFramework/ios/SomeFramework.framework.dSYM'),
+            Framework.new('${BUILT_PRODUCTS_DIR}/DebugCompiledFramework/CompiledFramework.framework', nil),
+            Framework.new('${PODS_ROOT}/ReleaseVendoredFramework/ios/SomeFramework.framework',
+                          '${PODS_ROOT}/ReleaseVendoredFramework/ios/SomeFramework.framework.dSYM'),
+            Framework.new('${BUILT_PRODUCTS_DIR}/ReleaseCompiledFramework/CompiledFramework.framework', nil),
+          ]
+          TargetIntegrator.framework_output_paths(paths).sort.should == %w(
+            ${DWARF_DSYM_FOLDER_PATH}/SomeFramework.framework.dSYM
+            ${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/CompiledFramework.framework
+            ${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/SomeFramework.framework
+          )
+        end
+
+        it 'calculates the output paths of the copy resources script' do
+          resource_paths = %w(
```

Comment:
I didn't change this logic but it didn't have a dedicated unit test so I added one

Reply:
could have sworn this existed was it only framework paths?

Reply:
there are tests but not for this individual function, they are more like integration tests

https://github.com/CocoaPods/CocoaPods/blob/ab5bd128bbd4a78328619993c3bbb76ef9304078/spec/unit/installer/user_project_integrator/target_integrator_spec.rb#L323-L363

Reply:
cool!